Source/CustomVoice.cpp

```diff
@@ -10,18 +10,31 @@
 
 #include "CustomVoice.h"
 
+// Indicates if this voice object is capable of playing the given sound.
+//
+// @param sound: A SynthesiserSound to be associated with this voice.
 bool CustomVoice::canPlaySound (juce::SynthesiserSound* sound)
```

Comment:
do we want an `@return` here for consistency? Even though it's not really necessary for clarity...

Reply:
Good call, added [here](https://github.com/HudsWilSu/Subsynth/pull/15/commits/433d1bd34b6f538d47b91eda5355e5324dcc68e6)